authorization-dotnet/TwoFactorAuthenticationGoogleAuthenticatorAngular/TwoFactorAuthenticationGoogleAuthenticatorAngular/DataTransferObjects/UserForRegistrationDto.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace TwoFactorAuthenticationGoogleAuthenticatorAngular.DataTransferObjects
+{
+    public class UserForRegistrationDto
+    {
+        public string? FirstName { get; set; }
+        public string? LastName { get; set; }
+
+        [Required(ErrorMessage = "Email is required.")]
+        public string? Email { get; set; }
+
+        [Required(ErrorMessage = "Password is required")]
```

Comment:
Please add a dot at the end of the sentence -> "Password is required.", just to be consistent with other annotation error messages on this class.